### PR TITLE
ThreatStream: Bug Fix - PAPP-26387: bug fix for whois domain action

### PR DESCRIPTION
**Pull request by ighelani-crest**
#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [x] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

### CODE REVIEW

**Review by rgil-splunk — COMMENTED** [review]
since the automatic pipeline fails due to the massive number of playbooks, could you link to a passing pipeline which covers these changes?

**Comment by spopat-crest** [comment]
Custom pipeline link: 
All tests: https://cd.splunkdev.com/phantom-apps/app-tests/-/jobs/39004920
Only cloud playbooks: https://cd.splunkdev.com/phantom-apps/app-tests/-/jobs/38506434

As there are many playbooks for the app and some of them are flaky, we have run two pipelines and the playbook failing in 1st pipeline is passing in the second one.

File: threatstream_connector.py [hunk 1]
```diff
@@ -515,7 +522,7 @@ def _whois(self, value, action_result, tipe=""):
             final_response.update(whois_response)
         except Exception as e:
             return action_result.set_status(phantom.APP_ERROR,
-                THREATSTREAM_ERR_PARSE_REPLY.format(error=self._get_error_message_from_exception(e)))
+                                            THREATSTREAM_ERR_PARSE_REPLY.format(error=self._get_error_message_from_exception(e)))
 
         try:
             if tipe == "ip":
```
Comment (splunk-jessica) [inline threatstream_connector.py:528]: Is this a typo for the word 'type'?

File: threatstream_connector.py [hunk 2]
```diff
@@ -1829,8 +1911,10 @@ def _handle_detonate_file(self, param):
             "report_radio-classification": param.get('classification')
         })
         if param.get("use_premium_sandbox", None) and param.get("use_vmray_sandbox", None):
-            return action_result.set_status(phantom.APP_ERROR,
-                "Both premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them.")
+            return action_result.set_status(
+                phantom.APP_ERROR,
+                "Both premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them")
```
Comment (splunk-jessica) [inline threatstream_connector.py:1916]: "Premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them")

File: threatstream_connector.py [hunk 3]
```diff
@@ -1886,8 +1971,8 @@ def _handle_detonate_url(self, param):
         })
 
         if param.get("use_premium_sandbox", None) and param.get("use_vmray_sandbox", None):
-            return action_result.set_status(phantom.APP_ERROR,
-                "Both premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them.")
+            error_msg = "Both premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them"
```
Comment (splunk-jessica) [inline threatstream_connector.py:1974]:
```suggestion
            error_msg = "Both premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them"
```

File: threatstream_connector.py (hunk 1 above)
Comment (ighelani-crest) [reply threatstream_connector.py]: Updated the variable name to "type".

File: threatstream_connector.py (hunk 2 above)
Comment (ighelani-crest) [reply threatstream_connector.py]: Included the suggested change.

File: threatstream_connector.py (hunk 3 above)
Comment (ighelani-crest) [reply threatstream_connector.py]: Included the suggested change.

**Review by splunk-jessica — APPROVED** [review]

**Review by ishans-crest — APPROVED** [review]

**Pull request merged**